src/components/customRoute/index.js

```diff
@@ -9,7 +10,9 @@ export const CustomRouteRender = ({ path, component, isPrivate, exact,
   const fullPath = pathPrefix + path + pathSuffix;
   return ((isPrivate && isAuthenticated) || !isPrivate ?
     <main className={isPrivate ? offlineStyle.disableWhenOffline : null}>
-      <Route path={fullPath} component={component} exact={exact} />
+      <ErrorBoundary errorMessage='An error occoured while rendering this page'>
```

Comment:
The`errorMessage` should have translation`t('...')` function. It might be easier if `errorMessage` is inside  `ErrorBoundary`.

Reply:
the translation key is passed as props, and errorBoundary uses t('') function on the passed props translation key. If we move it inside the boundary component we won't have the possibility to change the message for different use cases